cygnus-common/src/main/java/com/telefonica/iot/cygnus/backends/mongo/MongoBackendImpl.java

```diff
@@ -96,18 +96,6 @@ public void createCollection(String dbName, String collectionName, long dataExpi
         LOGGER.debug("Creating Mongo collection=" + collectionName + " at database=" + dbName);
         MongoDatabase db = getDatabase(dbName);
 
-        // create the collection
-        try {
-            db.createCollection(collectionName);
-        } catch (MongoException e) {
-            ErrorCategory errorCategory = ErrorCategory.fromErrorCode( e.getCode() );
-            if (errorCategory == ErrorCategory.fromErrorCode(48)){
-                LOGGER.debug("Collection already exists, nothing to create");
-            } else {
-                throw e;
-            } // if else
-        } // try catch
-
         // check STH indexes documentation at https://github.com/telefonicaid/fiware-sth-comet/blob/master/doc/manuals/db_indexes.md
```

Comment:
Maybe we could add this to the JavaDoc of this method:

> Note that different from the homonym createCollection method in this class, this one doesn't do an explicit collection creation operation in the DB, given that the creation of the indexes will automatically create the collection

as reminder for the future

Reply:
Done in https://github.com/telefonicaid/fiware-cygnus/pull/2164/commits/09057b15eabe233fc3ebd1b4a987f2bb38419d27